.github/workflows/release-artifacts.yaml

```diff
@@ -98,10 +99,20 @@ jobs:
             experimental-features = nix-command flakes
             accept-flake-config = true
           nix_path: "nixpkgs=channel:nixos-unstable"
+      - name: "Build static binary"
+        run: |
+          nix build --log-format raw-with-logs .#nickel-static
+          cp ./result/bin/nickel nickel-${{ matrix.os.architecture }}-linux
+      - name: "Upload static binary as release asset"
+        env:
+          GH_TOKEN: ${{ github.token }}
+          RELEASE_TAG: ${{ github.event_name == 'release' && github.event.release.tag_name || github.event.inputs.release_tag }}
+        run: |
+          gh release upload --clobber $RELEASE_TAG nickel-${{ matrix.os.architecture }}-linux
```

Comment:
This allows the GitHub CLI to overwrite the release artifact if it's already there. Mostly, this prevents annoying failures in case we need to rerun the workflow.

Reply:
Note that `$RELEASE_TAG` is a positional argument and `--clobber` is just a flag to allow the overwriting.

Reply:
Ok, thanks!